spec/requests/display_controller_spec.rb

```diff
@@ -0,0 +1,30 @@
+# frozen_string_literal: true
+require "rails_helper"
+
+RSpec.describe Brightcove::DisplayController do
+  before do
+    SiteSetting.brightcove_enabled = true
+    SiteSetting.brightcove_account_id = "987654321"
+    SiteSetting.brightcove_client_id = 123
+    SiteSetting.brightcove_client_secret = "abc"
+  end
+
+  let!(:video) { Brightcove::Video.create!(video_id: "12", secret_access_key: "abcd", state: "ready", api_request_url: "https://hello.world/video.mp4", callback_key: SecureRandom.hex) }
+
+  it "works" do
+    get "/brightcove/video/#{video.video_id}"
+    expect(response.status).to eq(200)
+  end
+
+  it "raises 404 for missing video" do
+    get "/brightcove/video/123"
```

Comment:
Unlikely but still, if there are enough spec that create `Video` models this might fail 😉 It's better to `destroy!` an existing video and use its id.

Reply:
The `123` here corresponds to the `video_id` column in the table, which is not the primary key, and is not auto-incremented. So I think we should be safe in this particular case 🤞 

Thanks for the review @CvX ❤️